### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by MahdiRezaeiDev**
### Personal portfolio project: create the desktop version
in this branch I have made the following changes:

- Updated media queries to support the desktop version.
- Update the readme file.
- updated background for the desktop version

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -90,14 +90,6 @@ picture {
...
 }
 
-/* Inherit fonts for inputs and buttons */
-input,
-button,
-textarea,
-select {
-  font: inherit;
-}
-
```
Comment (zieeco) [inline src/css/style.css:100]: **NOTE: This change is only applicable to the media query breakpoint that you've used**

- Please, kindly ensure the min-width value you've used is not less than `768px`. As you used 
`@media screen and (min-width: 40em) {` which is equivalent to `640px`, Please kindly increase it to either `48em` or higher to meet the project requirement as this is a core requirement to have your project approved.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @MahdiSohaily,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/css/style.css (hunk 1 above)
Comment (MahdiRezaeiDev) [reply src/css/style.css:100]: As there is not any other media query other than mobile and desktop using 40em and 48 will not differ.
buy the way I will change it to 48em

**Review by jasonfelice — APPROVED** [review]
Hi @MahdiSohaily, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
Good commit messages✔️
Descriptive Pull Request✔️
GitHub Flow followed✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**